android/build.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/bash
 # Build Keyman Engine Android and KMAPro
 
+# Set sensible script defaults:
+# set -e: Terminate script if a command returns an error
+set -e
+# set -u: Terminate script if an unset variable is used
+#set -u: Not set because of $RELEASE_OEM
```

Comment:
Is it ok to comment this out because `$RELEASE_OEM` isn't always set?

Reply:
I guess so, for now.